CODE_REVIEW.md

```diff
@@ -0,0 +1,72 @@
+# Code review requiements
+
+This document describe responsibilities of code reviewers when reviewing PRs.
+
+Pull request process is described in [contributing guide](./CONTRIBUTING.md#submitting-pull-requests)
+
+## Base checklist
+
+- All automated test steps pass (e.g. tests, lints, build)
+- PR title follows [commit conventions](CONTRIBUTING.md#commit-conventions)
+    - If PR format is different, reviewer should change it to follow the conventions
+- PR has a description with reasoning and change overview
+    - If description is missing but clear for reviewer, reviewer may add a comment or edit description
+- New feature or fix has tests proving it works
+    - Reviewer should request changes from contributor to add tests
+- If PR changes documented behaviour, there should be update to `/docs` or inline comments
+    - Reviewer should request changes from contributor to update docs
+- If PR introduces breaking changes, fixes a bug or adds a new feature, there should be a [release note](#release-notes)
+    - Reviewer may add a release note by themself
```

Comment:
I think we should clarify, whose responsibility it is to add the release note. I think it's PR author's responsibility. if we  not explicitly set the expectation here, one would think the other is going to do that.

Reply:
Good point. Added to contributors guide and mentioned here.